tests/models/test_initialization.py

```diff
@@ -0,0 +1,20 @@
+import pytest
+
+from vllm import LLM
+
+from .registry import HF_EXAMPLE_MODELS
+
+
+@pytest.mark.parametrize("model_arch", HF_EXAMPLE_MODELS.get_supported_archs())
+def test_can_initialize(model_arch):
+    model_info = HF_EXAMPLE_MODELS.get_hf_info(model_arch)
+
+    if not model_info.is_available_online:
+        pytest.skip("Model is not available online")
+
+    try:
+        LLM(model_info.default,
```

Comment:
would some models be too large to initialize in the ci's L4 GPU?

Reply:
Yeah, I am trying to find a solution to this...

Reply:
maybe adds some hf config override to make the model smaller. i do this when i test large models.

Reply:
Let me try setting `{"num_layers": 1, "num_hidden_layers": 1}`